content/blog/posts/reasons-to-go-to-grad-school.md

```diff
@@ -147,11 +147,11 @@ don't need a PhD to get an interesting job in industry, for whatever
 definition of "interesting" you want. Consider what you are going to do after
 you defend---do you need to have defended to have access the opportunities
 you want? There's certainly intrinsic value simply in finishing a PhD, but
-opportunity cost is also incredibly real, and the process can be [isolating,
+opportunity cost is also very real, and the process can be [isolating,
```

Comment:
You use "incredibly" at the end of the second paragraph after this, and I think it works better there.